src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -206,6 +206,17 @@ const USBankOnlyLabel = styled(Text)`
   color: ${themeGet("colors.orange150")};
   padding: 1px 5px;
   border-radius: 2px;
+  align-self: center;
+`
+
+const RadioWithLabel = styled(BorderedRadio)`
+  div {
+    flex-direction: row;
+    .jDlIOQ {
```

Comment:
**question:** I am unsure about using this class name as a CSS selector because it looks like a generated name, and I'm not sure if that will change over time. Is this a reliable class name to use for this purpose?